documentation/src/main/asciidoc/topics/con_server_directory_structure.adoc

```diff
@@ -0,0 +1,87 @@
+[id='server_directory_structure-{context}']
+= {brandname} Server Filesystem
+{brandname} Server uses the following folders on the host filesystem under `{server_home}`:
+
+[source,options="nowrap",subs=attributes+]
+----
+├── bin
+├── boot
+├── docs
+├── lib
+├── server
+└── static
+----
+
+== `server` folder
+
+Besides resources in the `docs` folder, the only folder you need to interact with is the `server` folder.
+
+[TIP]
+====
+See the _Server README_ for descriptions of the other folders in your `{server_home}` directory.
+
+The _Server README_ also describes system properties you can use to customize the {brandname} Server filesystem.
```

Comment:
How does a user get to the README? It seems like this should be linked here.

Reply:
Actually, why isn't this README just part of the normal docs?

Reply:
@ryanemerson It's linked in the `.Reference` section rather than inline. I have put the link inline but I think that takes users away from the topic. You can see it the current docs here: https://infinispan.org/docs/stable/titles/server/server.html#downloading_server_distributions 

Let me know what you think. If it's a better experience to go inline I'll change it.

Reply:
README is in the server folder so it's easier for engineering to maintain and contribute to. Docs as code. Admittedly I could do a better job of pulling this into the Server Guide rather than linking to github.

Reply:
It feels a little jarring jumping to GitHub from our docs. Where do we link to downstream?

Reply:
fixed! thanks for suggesting that improvement @ryanemerson